docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,142 @@
+
+    
+   
+
+2. Only `include` rules are specified 
+
+Cody is restricted from using content from repositories whose names match any specified patterns in the include field. Since no `exclude` rules are specified, no repositories are explicitly excluded
+
+<Callout type="note">An `include` field might contain a catch-all pattern (e.g., a `regexp` matching any string). In this case, Cody can access content from any repository, as a single match in the `include` rules is sufficient for inclusion.</Callout>
+
+    
+
+
+  
+   
+
+For example:
+
+```json5
+{
+	'cody.contextFilters': {
+		// Only repositories whose names either start with 'github.com/sourcegraph/'
+		// or contain 'cody' are allowed.
+		include: [
+			{repoNamePattern: '^github\\.com\\/sourcegraph\\/.+'},
+			{repoNamePattern: '.*cody.*'}
+		]
+	}
+}
+```
+
+3. Only `exclude` rules are specified
+
+All repositories are considered included by default without the' include' rules. Cody is prohibited from using content from any repository whose name matches at least one pattern specified in the exclude field.
+
+<Callout type="note">An `exclude` field may also contain a catch-all condition (e.g., a `regexp` matching any string). If such a match occurs, Cody is restricted from using content from any repository.</Callout>
+
+    
+
+
+
+For example:
+
+```json5
+{
+	'cody.contextFilters': {
+		// All repositories are allowed except for "github.com/sourcegraph/cody-analytics"
+		// or the ones containing "secret" in their name.
+		exclude: [
+			{repoNamePattern: '^github\\.com\\/sourcegraph\\/cody-analytics$'},
+			{repoNamePattern: '.*secret.*'}
+		]
+	}
+}
+```
+
+4. Both `include` and `exclude` rules are specified
+
+When making requests to third-party LLMs, Cody can use content from a repository if its name matches any of the `include` patterns
+and does not match any of the `exclude` patterns.
+
+Thus, the `exclude` rules filter only the repositories allowed by the `include` rules.
+
+For example:
+
+```json5
+{
+	'cody.contextFilters': {
+		// All repositories starting with "github.com/sourcegraph/" are allowed
+		include: [{repoNamePattern: '^github\\.com\\/sourcegraph\\/.+'}],
+		// except for "github.com/sourcegraph/cody-analytics"
+		// and the ones containing "cody" in their name.
+		exclude: [
+			{repoNamePattern: '^github\\.com\\/sourcegraph\\/cody-analytics$'},
+			{repoNamePattern: '.*cody.*'}
+		]
+	}
+}
+```
+
+## Cody Ignore File
+
+<Callout type="note">
+	Cody Ignore File functionality is available for all Cody Free, Pro and Enterprise users and is currently in the Experimental stage and only supported with
+	VS Code extension.
+</Callout>
+
+All Cody users can also configure the `.codyignore` file to specify files or folders from your codebase to be ignored as context
+by Cody through the following steps:
+
+### Enabling Unstable Features
```

Comment:
I feel like this "Enabling unstable features" section is a bit misleading, if i'm a user that has read about Cody Context Filters, and Cody Ignore files above, then it makes me feel like this section will teach me how to enable both features, which is not true as Cody Context Filters is not available yet and is not controlled through the editor. 

@MaedahBatool @aramaraju how should we clearly distinguish an experimental, unstable section (Cody Ignore), and a section that's clearly for Enterprise (Cody Context)?